venv/bin/activate

```diff
@@ -1,12 +1,18 @@
 # This file must be used with "source bin/activate" *from bash*
 # you cannot run it directly
 
+
```

Comment:
this means we are trying to add to the `venv` directory
let's add venv to your `.gitignore`